docs/logs/analysis-tab.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[xpack-logs-analysis]]
+== Detecting and inspecting log anomalies
+
+beta::[]
+
+With the {ml} {anomaly-detect} features enabled,
+you can use the *Log rate* page in the Logs app to automatically detect and inspect log anomalies and
+the log partitions in which they occurred.
+This helps you to spot suspicious behavior without significant human intervention --
+no more manually sampling log data, calculating rates, and determining if rates are normal.
+
+*Log rate* automatically highlights periods of time where the log rate is outside expected bounds and therefore may be anomalous.
+You can use this information as a basis for further investigations.
+For example:
+* A significant drop in the log rate might suggest that some some piece of infrastructure has stopped responding,
+and thus we're serving less requests.
+* A spike in the log rate could denote a DDoS attack.
+This may lead to an investigation of IP addresses from incoming requests.
+
+TIP: You can also view log anomalies directly in the <<xpack-ml-anomalies,Machine Learning app>>.
+
+[float]
+[[logs-analysis-create-ml-job]]
+=== Create a machine learning job for logs analysis
+
+To enable log analysis and anomaly detection, you must first create your own {kibana-ref}/xpack-spaces.html[space].
+Within a space, you can create a machine learning job which will carry out the logs analysis.
+
+[role="screenshot"]
+image::logs/images/analysis-tab-create-ml-job.png[Create machine learning job]
+
+1. Select the time range for the analysis.
+By default, the analysis uses logs no older than four weeks, and continues indefinitely.
+You cannot change the time range for the analysis after the machine learning job has been created.
+2. Add the indices containing the logs you want to analyze.
+3. Click *Create ML job*.
+
+Even though the analyze time range is fixed,
+you can still use the time filter to adjust the results that are shown.
+
+[float]
+[[logs-analysis-view-log-entries]]
+=== View log entries
+
+The *Log rate* page displays the following:
+
+* <<logs-analysis-entries-chart>>
+* <<logs-analysis-anomalies-chart>>
+
+[role="screenshot"]
+image::logs/images/analysis-tab.png[Analysis tab in Logs app in Kibana]
+
+[float]
+[[logs-analysis-entries-chart]]
+==== Log entries chart
+
+The log entries chart shows an overall visualization of the log entry rate,
+partitioned and color-coded according to the value of the Elastic Common Schema (ECS)
```

Comment:
Total nitpick: the partitioning is based on the ECS `event.dataset` field, but the color-coding follows the same severity colouring from the ML UI (and isn't directly linked to the partitioning itself).